ESPixelStick/src/input/externalInput.h

```diff
@@ -15,8 +15,6 @@ class c_ExternalInput
 	enum InputValue_t
 	{
 		off = 0,		// input is off
-		shortOn,		// input was on for 0.5 sec
-		longOn,         // input was on for 2.0 sec

```

Comment:
These values are for a feature not yet implemented and should be left in place.

Reply:
I would challenge you to reconsider the decision to leave unimplemented functionality in the code.  It makes the code harder to understand.  In addition, if any code implements a `switch()` on this enum value, it would need to explicitly list two cases which cannot occur ... or have build warnings.

If you really want to leave hints of unimplemented but future additions, consider at least rename the enums, such as `shortOn__NOT_YET_IMPLEMENTED` or `longOn__FUTURE_FUNCTIONALITY`?

Reply:
I have reverted the removal of these two unused enumeration values.